### PR TITLE
commands to downgrade the database schema and display version

### PR DESCRIPTION
**Pull request by ubozov**
This commit includes the following changes:

1.  `db-version` - command to display version of the database schema
2.  executing migration scripts to upgrade/downgrade database schema.

Resolves BV-858.

### CODE REVIEW

File: data/command.go [hunk 1]
```diff
@@ -13,39 +14,58 @@ import (
...
 
 	if *connStr == "" {
 		panic(errors.New("connection string is not detected"))
 	}
-	return *connStr
+
+	m := make(map[string]string)
+
+	m["conn"] = *connStr
+	m["version"] = strconv.Itoa(*version)
```
Comment (drew2a) [inline data/command.go]: The same as in previous review. 
From my perspective, if some string has been used more than 1 time, then better to extract it to const.

Comment (ubozov) [reply data/command.go]: updated

File: data/command.go [hunk 2]
```diff
@@ -5,47 +5,75 @@ import (
...
 		return nil
 	}
 
 	switch args[0] {
 	case "db-migrate":
-		connStr := getConnectionString(args)
-		if err := migration.Update(connStr); err != nil {
+		m := readFlags(args)
+		connStr := m[ConnectionString]
+		version, _ := strconv.ParseInt(m[Version], 10, 64)
```
Comment (furkhat) [inline data/command.go]: lets not ignore the error.

File: data/command.go [hunk 3]
```diff
@@ -5,47 +5,75 @@ import (
...
 	reform "gopkg.in/reform.v1"
 )
 
+type cmdFlag int
+
+// cmdFlags enum
+const (
+	ConnectionString cmdFlag = iota
+	Version
+)
```
Comment (furkhat) [inline data/command.go]: what if we define a struct and `readFlags` will return the instance of this struct? It gives better abstraction and encapsulation. 
This way we don't need those constants and type above.

File: data/command.go (hunk 2 above)
Comment (ubozov) [reply data/command.go]: updated

File: data/command.go (hunk 3 above)
Comment (ubozov) [reply data/command.go]: good idea, updated

**Pull request merged**